src/components/Tooltip/Tooltip.test.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import theme from 'theme';
+import { axe, mountWithTheme } from 'testUtils';
+import Tooltip from './Tooltip';
+
+describe('<Tooltip />', () => {
+  it('renders correctly', () => {
+    const render = () => mountWithTheme(<Tooltip>Save 45%</Tooltip>, theme);
+    expect(render()).toMatchSnapshot();
+  });
+
+  it('has no accessibility errors', async () => {
+    const render = () => mountWithTheme(<Tooltip>Save 45%</Tooltip>, theme);
+    expect(await axe(render().html())).toHaveNoViolations();
+  });
+
+  it('renders different pointing direction', () => {
+    const render = () =>
+      mountWithTheme(
+        <Tooltip pointingDirection="left">Save 45%</Tooltip>,
```

Comment:
Please test all values so all code paths are tested.

```suggestion
  it.each(['top', 'right', 'bottom', 'left'])('renders different pointing direction: %s', (direction) => {
```    const render = () =>
      mountWithTheme(
        <Tooltip pointingDirection={direction}>Save 45%</Tooltip>,
```

Reply:
🚀